lib/ui/pages/account-linking/account_unlinking.dart

```diff
@@ -0,0 +1,204 @@
+import 'dart:ui';
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:pispapp/controllers/ephemeral/account-linking/account_unlinking_controller.dart';
+import 'package:pispapp/models/consent.dart';
+import 'package:pispapp/models/currency.dart';
+import 'package:pispapp/repositories/firebase/consent_repository.dart';
+import 'package:pispapp/ui/theme/light_theme.dart';
+import 'package:pispapp/ui/widgets/shadow_box.dart';
+import 'package:pispapp/ui/widgets/title_text.dart';
+
+class AccountUnlinking extends StatelessWidget {
+  final AccountUnlinkingController _accountUnlinkingController =
+      AccountUnlinkingController(Get.find<ConsentRepository>());
```

Comment:
Why do we need to store the `ConsentRepository` globally? Would it be better to just pass it as a construction parameter for `AccountUnlinking`?

Reply:
I anticipate list of accounts being used in a couple of places. I don't want to fetch them every time they need to be used/displayed.

Reply:
I see, okay then if you think it's better that way :)